src/components/e-date.vue

```diff
@@ -10,15 +10,15 @@
       :class="b('input')"
       :maxlength="format.length"
       :name="name"
-      @keypress.enter="onEnter"
+      @keypress.enter.prevent
+      @keypress="onEnter"
```

Comment:
todo: sorry, this was probably misleading. You should extend the existing eventhandler, not add a new one.

```suggestion
      @keypress.enter.prevent="onEnter"
```